catroid/src/main/java/org/catrobat/catroid/utils/NumberFormats.java

```diff
@@ -40,15 +40,14 @@ public static String stringWithoutTrailingZero(String value) {
 		return value;
 	}
 
-	public static String humanFriendlyFormattedShortNumber(final int number) {
-		if (number < 1_000) {
-			return Integer.toString(number);
-		} else if (number < 10_000) {
-			return Integer.toString(number / 1_000) + (number % 1_000 > 100 ? "."
-					+ Integer.toString((number % 1_000) / 100) : "") + "k";
-		} else if (number < 1_000_000) {
-			return Integer.toString(number / 1_000) + "k";
+	public static String toMetricPostfixNotation(int number) {
```

Comment:
I think toMetricUnitRepresentation /Notation would be a better name since the Postfix notation is associated with reverse polish notation and calculators.

Reply:
Done, thanks!